cesql/spec.md

```diff
@@ -200,6 +200,8 @@ _Boolean_.
 When addressing an attribute not included in the input event, the subexpression referencing the missing attribute MUST evaluate to `false`.
 For example, `true AND (missingAttribute = "")` would evaluate to `false` as the subexpression `missingAttribute = ""` would be false.
 
+When addressing an attribute represented by the _Integer_ type, if the value of that attribute does not fit into a 32 bit signed integer, an error MUST be returned.
```

Comment:
A CloudEvent Integer attribute can't have such a value though... from the CE Spec:

> A whole number in the range -2,147,483,648 to +2,147,483,647 inclusive. This is the range of a signed, 32-bit, twos-complement encoding. Event formats do not have to use this encoding, but they MUST only use Integer values in this range.

To put it another way: if we've got this for integers, do we need something similar for attributes represented by the _Boolean_ type but with a value which isn't true or false?

Reply:
Ah, if it isn't possible for this value to be in an event then I think we can remove this - I hadn't been aware that that was the case

Reply:
Yes, it's the CE type system rather than about specific attributes.